eng/Versions.props

```diff
@@ -23,7 +23,7 @@
     <!-- Packages from dotnet/runtime -->
     <MicrosoftExtensionsHostingVersion>6.0.0</MicrosoftExtensionsHostingVersion>
     <!-- Packages from dotnet/sourcelink -->
-    <MicrosoftSourceLinkVersion>1.1.0-beta-20206-02</MicrosoftSourceLinkVersion>
+    <MicrosoftSourceLinkGitHubVersion>1.1.0-beta-20206-02</MicrosoftSourceLinkGitHubVersion>
```

Comment:
This file isn't included in a "real" build because dotnet/aspnetcore includes dotnet/spa-templates as a submodule from 6.0.x onwards. The change makes sense when people build spa-templates locally but doesn't affect any CI; this repo has no associated pipelines except for its `OneLocBuild` one. I'm not sure that matters however and changing this will certainly avoid breaking local builds.

Reply:
That probably means this property can just be eliminated.

Reply:
Yep, updating the property name still sounds correct.